lets_play/python_snippets.py

```diff
@@ -0,0 +1,32 @@
+class CoolClass:
+    def __init__(self):
+        self.coolness = 10
+
+    def increaseCoolness(self, delta):
+        self.coolness += delta
+
+    def __str__(self):
+        return "I am " + str(self.coolness) + " cool!"
+
+    def fun():
+        if 1:
+            if [1]:
+                if True:
+                    if "some_string":
```

Comment:
Keep in mind that `0`, `[]`, `False`, and `""` will all evaluate to false (that is, the `if` case will not be taken).